tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestExampleGenerator.java

```diff
@@ -138,12 +138,14 @@ public void testCurrency() {
         "//ldml/numbers/currencyFormats/currencySpacing/afterCurrency/currencyMatch",
         "//ldml/numbers/currencyFormats/currencySpacing/afterCurrency/surroundingMatch",
         "//ldml/numbers/currencyFormats/currencySpacing/afterCurrency/insertBetween",
+        "//ldml/numbers/currencyFormats/currencyPatternAppendISO", // TODO see CLDR-14831
```

Comment:
CLDR-14831 doesn't mention anything about currency formatting.

Reply:
Yes, that is in the linked ticket https://unicode-org.atlassian.net/browse/CLDR-15286